multiregion/docs/multiregion.rst

```diff
@@ -1243,23 +1279,51 @@ it is possible Docker networking not working or cleaning up properly between run
 
 
 .. |Multi-region Architecture|
-   image:: images/multi-region-base-v2.png
+image:: images/multi-region-base-v2.png
    :alt: Multi-region Architecture
 
 .. |Follower_Fetching|
-   image:: images/Follower_Fetching.png
+image:: images/Follower_Fetching.png
    :alt: Follower fetching
 
 .. |Multi-region latencies|
-   image:: images/multi-region-latencies-v2.png
+image:: images/multi-region-latencies-v2.png
    :alt: Multi-region latencies
 
 .. |Multi-region topic replicas|
-   image:: images/multi-region-topic-replicas-v2.png
+image:: images/multi-region-topic-replicas-v2.png
    :alt: Multi-region topic replicas
 
+.. |C3 cluster navigation|
+image:: images/c3-cluster-navigation.png
+   :alt: C3 cluster navigation
+
+.. |C3 topics overview|
+image:: images/c3-topics-overview.png
+   :alt: C3 topics overview
+
+.. |multi-region-async replicas|
+image:: images/multi-region-async-placement.png
+   :alt: multi-region-async replicas
```

Comment:
@awalther28 WDYT about adding the following to make the graphic smaller?

```
      :width: 500px
```

Reply:
The smaller picture looks much better. 
<img width="1108" alt="Screen Shot 2021-01-04 at 11 56 39 AM" src="https://user-images.githubusercontent.com/12937975/103569093-eedbda00-4e83-11eb-84d4-ad78f22732f7.png">